src/test/java/core/basesyntax/StorageDaoTests.java

```diff
@@ -0,0 +1,126 @@
+package core.basesyntax;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class StorageDaoTests {
+    private static final int MIN_AGE = 18;
+    private static final int MIN_LENGTH = 6;
+    private static RegistrationService registrationService;
+    private static User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        user = new User();
+    }
+
+    @BeforeEach
+    void setUp() {
+        Storage.people.clear();
+        user.setAge(21);
+        user.setLogin("mateAcademy");
+        user.setPassword("mateAcademy2021");
+    }
+    
+    @AfterEach
+    void tearDown() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_notExistedUser_ok() {
+        assertEquals(user,registrationService.register(user));
+    }
+
+    @Test
+    void register_validAge_ok() {
+        assertTrue(registrationService.register(user)
+                .getAge() >= MIN_AGE);
```

Comment:
let's not make this check and use assertEquals instead of assertTrue and we could check the user that comes after execution of register() with user that you pass as parameter